game/src/gui/app/equipment_select_view.h

```diff
@@ -19,19 +19,24 @@ namespace gui {
 namespace app {
 
 class EquipmentSetView;
+class GameView;
 
 class ItemIconView : public CallbackView {
  public:
-  ItemIconView(const Rect&, const core::EquipmentWithAmount&);
+  ItemIconView(GameView*, const Rect&, const core::EquipmentWithAmount&);
```

Comment:
I would like to keep `const Rect&, ` as the first param, since all other View classes are of that convention. Could you please move `GameView*` to be the second or third param?

Reply:
OK. I got it :)